pages/edge/tutorials/_meta.json

```diff
@@ -1,4 +1,5 @@
 {
   "react-static-site": "React Static Site",
-  "python-flask-server": "Python Flask Server"
+  "python-flask-server": "Python Flask Server",
+  "cdn-on-wasmer-edge": "CDN on Wasmer Edge"
```

Comment:
This should be `Wasmer Edge as CDN` (and the url should be just `cdn`)

Reply:
changed that